<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.23">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Jason Collins">
<meta name="dcterms.date" content="2025-05-28">

<title>Economists’ Genetic Blindspot: The Data We’re Not Collecting – Jason Collins blog</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js" type="module"></script>
<script src="../site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-226bd0f977fa82dfae4534cac220d79a.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap-ac706bccb5107cfdbcdabb2cff0da72d.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script async="" src="https://www.googletagmanager.com/gtag/js?id=G-C50MEPDMZ9"></script>

<script type="text/javascript">

window.dataLayer = window.dataLayer || [];
function gtag(){dataLayer.push(arguments);}
gtag('js', new Date());
gtag('config', 'G-C50MEPDMZ9', { 'anonymize_ip': true});
</script>


<link rel="stylesheet" href="../styles.css">
<meta property="og:title" content="Economists’ Genetic Blindspot: The Data We’re Not Collecting – Jason Collins blog">
<meta property="og:description" content="Behavioural economics, data science and artificial intelligence.">
<meta property="og:site_name" content="Jason Collins blog">
<meta name="twitter:title" content="Economists’ Genetic Blindspot: The Data We’re Not Collecting – Jason Collins blog">
<meta name="twitter:description" content="Behavioural economics, data science and artificial intelligence.">
<meta name="twitter:card" content="summary">
</head>

<body class="nav-fixed fullcontent quarto-light">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">Jason Collins blog</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="../about.html"> 
<span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../research.html"> 
<span class="menu-text">Research</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../subscribe.html"> 
<span class="menu-text">Subscribe</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../teaching.html"> 
<span class="menu-text">Teaching</span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="../index.xml"> <i class="bi bi-rss" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Economists’ Genetic Blindspot: The Data We’re Not Collecting</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Jason Collins </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">May 28, 2025</p>
    </div>
  </div>
  
    
  </div>
  


</header>


<p>This article was intended for a different forum. When that didn’t work out, I decided to park it here.</p>
<p>–</p>
<p>Professor of Sociology William H. Sewell was deeply interested in social mobility. Do career aspirations affect career achievement? Do individual and social traits underlie those aspirations? Despite some preliminary research in the 1950s, Sewell lacked the data to answer these questions.</p>
<p>In 1962, Sewell <a href="https://doi.org/10.1016/S0276-5624%2803%2920001-9">made a lucky discovery</a>: sitting unused in a University of Wisconsin administration building were the survey schedules and punch cards from a 1957 survey on the educational plans of all Wisconsin high school seniors. Now he had what he needed. He randomly selected 10,317 of these seniors and, in 1964, sent postcard surveys to their parents, asking about the seniors’ education, career aspirations, and socioeconomic status. Eighty-seven per cent of the parents responded. A 1975 telephone survey of the graduates themselves had a response rate of 89 per cent.</p>
<p>These steps began the Wisconsin Longitudinal Study; longitudinal studies observe subjects over time. In 1977, the survey expanded to include randomly selected siblings of the graduates. By 2020, the survey had accumulated over 60 years of data. The high school graduates were 81 years old.</p>
<p>In 2006-07 was a significant milestone. Saliva sample kits for genetic analysis were posted to participants. Kits were mailed in 2011 to those who missed the first round. Overall, 64 per cent of graduates and 36 per cent of siblings provided samples, covering about half of the 18,000 participants.</p>
<p>As genetic data doesn’t change, these samples enriched 50 years of data, including that initial collection in 1957. Each new collection, such as those in 2011 and 2020, is also augmented by this genetic data.</p>
<p>In 2018, Daniel Belsky and colleagues published <a href="https://doi.org/10.1073/pnas.1801238115">a paper</a> in the Proceedings of the National Academy of Sciences using the Wisconsin Longitudinal Study’s genetic data. For each student, they calculated an education “polygenic score”, a measure of the genetic influence on educational attainment. Students with higher scores had higher career success and social rank than their parents did in 1957. They were upwardly mobile. This observation held within families: siblings with higher scores achieved more as well.</p>
<p>As genetics are the cause of many phenomena we study, genetic data can be of immense value. Studying the mechanisms of DNA transmission and recombination between generations can help policymakers investigating issues such as social mobility, poverty, and inequality. We could assess interventions, from education to tax reform to childcare. By learning which interventions work, we could allocate public resources more efficiently. Kathryn Paige Harden makes the case for the value of genetic analysis comprehensively in the excellent <a href="https://press.princeton.edu/books/hardcover/9780691190808/the-genetic-lottery">The Genetic Lottery: Why DNA Matters for Social Equality</a>. In what follows, I will remake that case in only the most superficial way.</p>
<p>However, there is a major barrier to using genetic data in this way: most of the datasets we use to investigate socio-economic phenomena do not include genetic data, preventing us from including genetics in our analyses.</p>
<p>We should address this problem by collecting genetic data from the participants in longitudinal research. As our DNA is fixed for life, we should supplement old longitudinal data with genetic data, enriching decades of past work. The addition of genetic data to the Wisconsin Longitudinal Survey provides a template.</p>
<p>We could also be bringing genetic data to bear in our experimental work. The addition of genetic data to experimental panels could provide rich insight into the heterogeneity of behaviour.</p>
<p>Importantly, we can do this now. While many discussions on genetic data in social science focus on growing sample size, new tools and future possibilities, existing tools can give us insight today and bring future possibilities to life.</p>
<p>In what follows, I will first touch on two topics of interest to economists (my own profession): using genetic transmission to infer causation and some examples of genetic data applied to questions of social mobility and inequality. That will take me to my main point, that we should be supplementing our core research datasets with genetic data now.</p>
<section id="causation" class="level1">
<h1>Causation</h1>
<p>There is abundant data indicating the intergenerational persistence of educational outcomes and socioeconomic status. Here are two Australian examples (I’ll lean on material from my home country, which I know best): A Year 9 student (aged 14-15) whose parents have a bachelor’s degree or higher will, on average, have numeracy skills almost <a href="https://www.pc.gov.au/inquiries/completed/school-agreement/report/school-agreement.pdf">four years ahead of those of classmates</a> whose parents do not have a bachelor’s degree. A child born in Australia to a family in the bottom 20 per cent of parental incomes <a href="https://doi.org/10.1016/j.labeco.2020.101861">has a 12 per cent chance</a> of being in the top 20 per cent of incomes 30 years later.</p>
<p>For an economic policymaker, these statistics raise questions. Do the children of educated, wealthy parents have an unfair advantage? Would equalising wealth through taxes and transfers close the intergenerational gap? Designing robust solutions requires accurately assessing causality, but a correlation between child and parent outcomes does not prove that higher parental education or socioeconomic class <em>causes</em> outcomes in the next generation. We need to consider if a third factor might be “confounding” the result.</p>
<p>A likely third factor is genetics. Children and parents share DNA. If parents genetically transmitted traits like intelligence and conscientiousness to their children, we could see a correlation between the child and parent even if parental education or income had no direct effect on the child. It should not be controversial to say that genetics <em>could</em> underlie this result: the <a href="https://doi.org/10.1111/1467-8721.00084">first law of behaviour genetics</a> is that all human behavioural traits are heritable.</p>
<p>The genetic confound raises the question of how to infer the cause. Economists are infatuated with causation; absent a randomised controlled trial, they scour the world for interesting data sets and quasi-experiments to tease out causality. This pursuit led to innovative approaches to infer causation. Economists celebrate the “<a href="https://en.wikipedia.org/wiki/Credibility_revolution">credibility revolution</a>”, demanding rigorous study design.</p>
<p>An example quasi-experiment concerns <a href="https://en.wikipedia.org/wiki/May_68">protests in Paris in May 1968</a>, which led authorities to be lenient in university entrance exams. <a href="https://doi.org/10.1086/522071">Eric Maurin and Sandra McNally studied</a> students who barely passed despite the increased leniency, and likely would not have been accepted in other years. These students earned higher future wages, and, in turn, their children obtained more education. As the riot did not affect the genetics of the parents, we can take the change in schooling as the cause.</p>
<p>Economists’ focus on causation does have a benefit. The social science literature is littered with studies for which genetics is an obvious confound. There are fewer examples in the economics literature (that is my impression, at least). Although economists rarely discuss genetics, they prefer experimental designs that avoid confounds. That, however, places a constraint on the questions that we can answer. Thankfully, overcoming the genetic confound does not always require a city to descend into riots.</p>
<hr>
<p>Human genomes consist of 3 billion base pairs, with over 99 per cent of base pairs the same from person to person. Most of the variation between people are single nucleotide polymorphisms (SNPs), changes in just one base pair that are present in at least 1 per cent of the population. Genetic databases, like the Wisconsin Longitudinal Study, typically contain samples of SNPs. If you’ve taken a DNA test from companies like 23andMe (R.I.P.) or MyHeritage, they analysed your SNPs too.</p>
<p>Research using SNP data suggests that most traits are polygenic: that is, many genes underlie the traits’ heritability. For example, one study <a href="https://doi.org/10.1038/s41588-022-01016-z">identified 3,952 SNPs</a> linked to educational attainment. This reflects a proposed <a href="https://doi.org/10.1177/0963721415580430">fourth law of behaviour genetics</a>: ‘A typical human behavioural trait is associated with very many genetic variants, each of which accounts for a very small percentage of the behavioural variability.’</p>
<p>A long list of SNPs alone is not useful for examining social science outcomes. As a result, scientists developed polygenic scores, a weighted count of SNPs enhancing a trait. To give an example, <a href="https://doi.org/10.1177/0956797616643070">one study</a> found that a person in the 84th percentile of polygenic scores for educational attainment was 19 per cent more likely to complete a university degree than someone with an average score. It may not sound like much, but this is a similar effect size to that of many social and cultural factors, like family socioeconomic status.</p>
<p>Despite the apparent link between polygenic scores and outcomes, this is again correlation and not proof of causation. Does a link between polygenic score and outcome mean genetics <em>caused</em> the difference? We cannot jump to an answer of “yes”. Population stratification may be at play, where genetic differences arise between groups due to historical, cultural and social factors. For example, intermarriage among highly educated groups can lead to genetic variant concentration over time, showing genetics to be a historic contingency and not truly causal. We can at least rule out reverse causation; education does not alter DNA.</p>
<p>However, the nature of DNA transmission from parent to child provides a mechanism by which we can get closer to the cause. Your genetic material is organized into 23 pairs of chromosomes, one-half of each pair coming from each of your father and mother. Each parent’s chromosomes in turn came from your grandparents, but you didn’t receive exact copies of your grandparents’ chromosomes. During the creation of your parents’ eggs and sperm, your grandparents’ chromosomes were spliced together, resulting in each egg or sperm having different combinations of your grandparents’ chromosomes. Women create about 45 splices, and men create about 26. The result is that siblings receive a random draw from each parent’s chromosomes. This draw involves <a href="https://doi.org/10.1038/ng917">a small number of chunks</a> (around 23+45=68 from the mother and 23+26=49 from the father), not each of the three billion base pairs independently. Because of this small number of chunks, the genetic relatedness between siblings can vary significantly from the average of 50 per cent, with most siblings sharing between <a href="https://doi.org/10.1371/journal.pgen.0020041">43 per cent and 57 per cent</a> of their genetic material. My identical twin sons share only 41 per cent of their DNA with their younger brother.</p>
<p>Understanding how DNA is transmitted provides a fantastic opportunity. Historically, behaviour genetics relied on comparing identical and fraternal twins or examining adoptees. Relatedness data now allows an <a href="https://doi.org/10.1371/journal.pgen.0020041">extension of the twin study methodology</a> to families without twins. The lower variation in relatedness between siblings (as compared to identical versus fraternal twins) reduces the ability to link between genes and outcomes, but this is offset by the larger samples available when you are no longer constrained to twin samples. <a href="https://doi.org/10.1038/s41588-018-0178-9">Researchers have since expanded this methodology</a> to include broader population samples, not just siblings, to estimate the heritability of traits like educational attainment.</p>
<p>Within-family variation in SNPs also allows us to calculate more robust polygenic scores. Since each sibling’s set of SNPs is the outcome of a lottery, causation is <a href="https://doi.org/10.1371/journal.pbio.3002511">less susceptible to bias</a>.</p>
<p>Another opportunity arises because some genetic variants are not passed from parent to child. If these non-transmitted variants affect child outcomes, we can rule out genetic transmission and focus on environmental channels such as socioeconomic status. <a href="https://doi.org/10.1038/s41562-020-0862-5">Kathryn Paige Harden and Philipp Koellinger</a> call this a ‘virtual parent’ design, which mimics adoption studies in that the children are raised by someone with different genes. <a href="https://doi.org/10.1126/science.aan6877">Kong and colleagues</a> used this premise to show that the effect of the non-transmitted variants on child education was 30 per cent as strong as the transmitted polygenic score.</p>
<p>The methodologies underlying this research are still in development and subject to some interesting debates. Are we <a href="https://www.google.com/search?q=https://doi.org/10.1101/2024.10.01.24314703">effectively controlling for population stratification</a> when we don’t have family-based samples? However, that is not a barrier to building data now, and further data collection will support resolving such debates.</p>
</section>
<section id="illustrating-the-applications" class="level1">
<h1>Illustrating the applications</h1>
<p>For all of the evidence indicating their influence, genes are not destiny. An example is <a href="https://doi.org/10.2307/2553675">Arthur Goldberger’s</a> thought experiment about eyeglasses. Poor eyesight due to genetics might be corrected by an environmental intervention.</p>
<p>Although Goldberger made this point to argue against using heritability in policy development, studies of intergenerational status have shown genes and environment are interconnected. Methodologies such as those described above can help us understand the causes of transmission, explore what policy interventions are most prospective and examine the distributional effects of those policies.</p>
<p>This article is not a comprehensive review of policy relevant research, but below are some brief illustrations related to social mobility and intergenerational transmission of socioeconomic status to give a flavour of the questions that can be illuminated with genetic data.</p>
<hr>
<p>What is the optimal level of social mobility?</p>
<p>I don’t know the answer to that question, but any answer requires us to consider genetics. With genetic endowments, random sorting will not emerge in a society with equal environments. Consider the following. A <a href="https://doi.org/10.1007/s10888-019-09413-x">study of Finnish twins</a> found a substantial genetic effect on lifetime earnings: around 40 per cent of the variance in women’s earnings and around half for men’s. Twenty-one other studies in Australia, Sweden and the United States produced similar estimates for genetic contribution, with the effect of shared environment, comprising common environmental factors such as parental socioeconomic status, being around 9 per cent. This is the <a href="https://doi.org/10.1111/1467-8721.00084">second law of behaviour genetics</a> in action: the effect of being raised in the same family is smaller than the effect of genes. However, is heritability capturing an inherent characteristic of the child, the parental response to the child’s genotype, or the environment created due to the parental genotype that is also shared with the child?</p>
<p>Analyses using genetic data are nascent, but they can shed light on this question. Polygenic scores for education are <a href="https://doi.org/10.1073/pnas.1801238115">linked to higher socioeconomic status</a> and <a href="https://doi.org/10.1093/jeea/jvz072">better labour outcomes</a>, even among siblings. However, those who grow up in high-status households tend to have higher college completion and better socioeconomic outcomes independent of their score. Further, there is a stronger relationship between polygenic scores and college completion in higher socioeconomic groups. While this pattern may partly reflect unobserved genetic variation - polygenic scores capture only some of the heritability of traits - this evidence suggests an opportunity to improve outcomes for talented students in low-status families.</p>
<p>There is growing research into the transmission of skills, one of the pathways by which socioeconomic status might persist. <a href="https://doi.org/10.1257/aer.20220456">One study</a> identified three genetic pathways: the direct genetic effect, whereby both parent and child have genes that increase their skills; parental investment in children with higher polygenic scores; and parents with higher genetic factors themselves investing more in their children. This study indicated that ignoring genetics overestimates the effect of parental investment on child skills, but that the environment created by the parent, influenced by their genetics, also matters, at least for the children aged seven years or less examined in this study. Examining these investments may provide intervention ideas.</p>
<p>Another study used genetic data to provide insight into the accumulation of wealth. People with a higher polygenic score for years of schooling <a href="https://doi.org/10.1086/705415">had greater household wealth at retirement</a>. Those with scores in the 84th percentile had over $150,000 extra wealth. A polygenic score premium persisted even after controlling for education and income, suggesting the score captures other skills. One policy insight comes from a gene-environment interaction. The relationship between wealth and polygenic score was four times as large for those without defined benefit pensions, which involve a guaranteed income and require few decisions about money allocation. Those with lower polygenic scores tended to struggle with managing their retirement investments, indicating that freedom can harm those who find complex financial decisions difficult. This finding could inform policy. In Australia, compulsory defined accumulation plans divert a portion of income into tax-advantaged retirement accounts that cannot be accessed until retirement. This is an ‘eyeglasses’ solution to the problem, albeit everyone gets eyeglasses regardless of need. A more targeted response might be Australia’s relatively generous means-tested pension system.</p>
<p>Genetic factors may also illuminate the distributional effects of policy. Taxes on tobacco were intended to curb usage, but as Jason Fletcher argued in a <a href="https://doi.org/10.1371/journal.pone.0050576">speculative article</a>, genetically disadvantaged populations might bear a higher burden. Variants of nicotine receptor genes may trigger different responses; those with higher reward responses in the brain didn’t reduce smoking despite the higher cost. This suggests that diversion through nicotine substitutes may be more effective than taxes for some populations.</p>
<p>Examples of heterogeneous genetic responses to policy interventions are growing. Raising the minimum school leaving age <a href="https://doi.org/10.1073/pnas.1802909115">reduced the body mass index</a> of those genetically prone to obesity. Students with low polygenic scores attending <a href="https://doi.org/10.1038/s41539-020-0060-2">advantaged schools</a> were less likely to drop out of math classes. Similarly, <a href="https://doi.org/10.1093/jeea/jvz072">the relationship</a> between polygenic scores and high school dropout was weaker in higher socioeconomic families, suggesting an ability to buffer against the worst outcomes. A useful heuristic to consider whether responses might vary with genetics is to consider whether they vary with education, income, or other socioeconomic factors. Genetics likely underlies many of these non-genetic variables.</p>
<p>The small but growing body of research provides a consistent picture: genetics influence social mobility and interact with environmental factors in non-obvious ways. Absent genetics, we struggle to assess the causes and overlook insights that can help us evaluate interventions.</p>
</section>
<section id="the-practical-steps" class="level1">
<h1>The practical steps</h1>
<p>Despite the above examples, there are many unexploited opportunities to use genetic data in economics and policy development. The literature is rich but small.</p>
<p>However, to exploit these opportunities, we need to enhance our datasets with genetic data. We need to build the genetic infrastructure. When economists plan research using longitudinal datasets or experimental panels, they shouldn’t have to think about how to collect saliva samples or the cost of genetic analysis. The genetic data should be there by default.</p>
<section id="enhancing-longitudinal-data" class="level3">
<h3 class="anchored" data-anchor-id="enhancing-longitudinal-data">Enhancing longitudinal data</h3>
<p>Longitudinal data sets are valuable for their multidimensionality over time, allowing researchers to track changes and examine the participants’ life paths. Ask an applied economist about their most valuable research resource, and they will often point to longitudinal data.</p>
<p>In Australia, our best-known longitudinal dataset is the Household, Income and Labour Dynamics in Australia (HILDA) Survey. Begun in 2001 with a sample of <a href="https://www.dss.gov.au/about-the-department/longitudinal-studies/living-in-australia-hilda-household-income-and-labour-dynamics-in-australia-overview">over 7,000 households</a>, 22 years of data are now available to researchers. <a href="https://melbourneinstitute.unimelb.edu.au/hilda/publications/journal-articles">Over 1,000 papers</a> have been published using HILDA data on topics ranging from <a href="https://doi.org/10.1007/s00148-017-0667-7">how losing a job affects who does chores at home</a> to the <a href="https://doi.org/10.1016/j.jebo.2015.02.004">effect of stock market performance on well-being</a> to <a href="https://doi.org/10.1111/1475-4932.12641">intergenerational social mobility</a>.</p>
<p>In the United States, perhaps the best-known longitudinal surveys are the <a href="https://en.wikipedia.org/wiki/National_Longitudinal_Surveys">National Longitudinal Surveys</a> (NLS) sponsored by the U.S. Bureau of Labour Statistics. These include the <a href="https://www.nlsinfo.org/">National Longitudinal Study of Youth</a> 1979 (tracking subjects born between 1957 and 1964) and 1997 (tracking subjects born between 1980 and 1984). Named for their starting years, both studies are still running today.</p>
<p>What makes these longitudinal resources valuable is the sheer depth of the surveys across topic areas and time. HILDA includes data on ancestry, children, education, finance, health, housing, labour force outcomes, skills and relationships. The challenge, however, is that even with the richness of the resource, it can take some ingenuity to infer what is causing the observed outcomes. Researchers use mechanisms such as the ordering of events, assuming causation must flow forward through time. But this typically has severe constraints.</p>
<p>Again, genetics provides a potential tool. If seeking to examine how parent behaviour affects child outcomes over time, genetic data could be used to disentangle environmental and genetic causes, and where genetic, examine the pathways by which the genetics operate. Even where genetic data does not identify causation, it might provide insight.</p>
<p>Unfortunately, many of our most valuable longitudinal datasets do not have associated genetic data. There are a few exceptions, including the Wisconsin Longitudinal Study and others that underlie the examples in this article. The <a href="https://www.teds.ac.uk/">Twins Early Development Study</a>, which has been tracking 15,000 twins born in the UK between 1994 and 1996, is now augmented with genetic data. The Dunedin study and the National Longitudinal Survey on Adolescent Health also contain genetic data. There are others, some of which underlie the examples in this article.</p>
<p>For every longitudinal dataset, we should augment data collection with genetic data. This does not apply only to new surveys. We could augment existing surveys with that data. Genetic data obtained from participants today could be used to analyse Round 1 HILDA or National Longitudinal Survey data. There are many studies where people have been tracked for decades where participants are still available. If genetic data collection is extended to families, many of the approaches to causation described in this article become available. The Wisconsin Longitudinal Study is an example of this occurring.</p>
<p>The nature of longitudinal surveys makes genetic sampling prospective. Longitudinal surveys have the benefit of a strong relationship with the participants. Participants already provide much sensitive information and may be willing to contribute genetic data. Most longitudinal surveys already have strong privacy protocols in place, controlling access to data based on sensitivity. Similar privacy measures can be applied to genetic data, addressing concerns such as the use of the genetic data to identify individuals. Instead of making the SNP data directly available, a set of polygenic scores and relatedness data within families could be released. If researchers need access to more detailed data, the existing access controls for the more sensitive longitudinal data provide prior art. The Wisconsin Longitudinal Study has such tiered access, with more stringent applications for SNP data than for aggregated polygenic scores.</p>
<p>The sheer volume of research from the major longitudinal surveys makes sampling highly cost-effective. With costs comfortably below $100 per person, the cost of genotyping for, say, the NLS Youth Surveys or HILDA is less than $1 million each. Sample collection can be done by post. The thousands of papers using this data, not to mention its use in government and by policymakers, make this a high-value step.</p>
<p>The cost-effectiveness will only increase. Genotyping and sequencing are becoming cheaper. Soon genetic samples will expand to the whole genome, capturing more rare variants and mitigating problems of population stratification. The polygenic scores that can be constructed with the genetic data will only increase in power.</p>
</section>
<section id="enhancing-experimental-data" class="level3">
<h3 class="anchored" data-anchor-id="enhancing-experimental-data"><strong>Enhancing experimental data</strong></h3>
<p>While most of the above discussion has been focussed on research using longitudinal datasets, there is also an opportunity to use genetic data to increase insight from experimental work.</p>
<p>Historically, students have been the typical subjects in economics or psychology experiments. They’re cheap and widely available on university campuses. This reliance on a narrow population slice led to an <a href="https://doi.org/10.1017/S0140525X0999152X">inevitable (at least in hindsight) critique</a>. Behaviour varies across populations. The standard experimental subject - someone Western, Educated, Industrialised, Rich and Democratic (WEIRD) - is not representative. The resulting data skew exists both <a href="https://doi.org/10.1257/aer.91.2.73">across populations or societies</a> and within them. College-educated students are not representative of those without a college education. The same holds for the rich and the poor.</p>
<p>One solution is to broaden subject pools. Today, experiments are less often conducted with students (although it’s unclear whether experimental participants sourced through Amazon’s crowdsourcing platform Mechanical Turk are more representative of humanity).</p>
<p>Another approach has been to examine how participants’ responses vary within experiments. Behavioural economists often point to <a href="https://doi.org/10.1038/s41562-021-01143-3">heterogeneity</a> as the future of applied behavioural science. People vary in capabilities, resources, goals and preferences. As a result, behavioural economists need to move beyond a one-size-fits-all philosophy and use <a href="https://doi.org/10.1017/bpp.2020.7">personalised nudges</a>. Practically, researchers address this by collecting data on gender, income, education and other demographic variables and studying how responses vary with demographic differences.</p>
<p>Genetics may enhance our understanding of heterogeneity. Per the first law of behavioural genetics, genetics may drive variability in experimental behaviour. Further, many characteristics that we identify as a source of variation may not capture the underlying cause. Genetic data can help us determine whether wealthy people respond differently because they are richer, or because they have characteristics that tend to lead to wealth. Genetic data offers a way to check that randomised controlled trials are balanced - that is, to test the assumption that each group is the same. It also allows us to increase the power of the analysis - our ability to detect an effect - by accounting for some of the variation between the experimental participants.</p>
<p>Integrating genetic data into experiments can let us investigate associations discovered in observational data. For example, the link between polygenic scores for <a href="https://doi.org/10.1086/705415">educational attainment and wealth at retirement</a> may be due to differences in the ability to make complex decisions. We could investigate that hypothesis through experiments examining how complex decision-making skills vary with the polygenic score. Experiments could provide a test bed for interventions, or at least inform policy discussions.</p>
<p>Enabling the use of genetics in experimental work requires building panels - collections of people who have registered to participate in experiments - with genetic data available for each participant. Each panel member could be genotyped, with the experimenter provided with polygenic scores for a range of outcomes of interest for each participant. The aggregated nature of these scores makes reidentification near impossible. As the effect sizes associated with polygenic scores are typically as strong or stronger than those for many social science interventions, genetic effects could be detected in experiments with as <a href="https://doi.org/10.1038%2Fs41588-018-0147-3">few participants as the typical lab experiment</a>.</p>
<p>A version of this has been done in the past with panels of twins, although in that case the genetic data is typically limited to whether the twins are mono- or dizygotic. For example, <a href="https://twins.org.au/research/research-with-us/">Twins Research Australia</a> maintains a panel of 35,000 twin pairs, which researchers may apply to access existing data or run a new study. One team of researchers developed the <a href="https://doi.org/10.26193/TTQEBQ">Australian Twins Economic Preferences Survey</a> using that data.</p>
<p>Incorporating genetic data into experimental work may be a bigger challenge than for longitudinal data, as there are no ready-made data collection and access arrangements. These would require investment. However, genotyping or sequencing cost are unlikely to be prohibitive. As panel participants typically participate in many experiments, the cost of the genetic tests could be spread over them. Since genetic data doesn’t change - unlike other attributes measured in experimental studies - it only needs to be collected once. Further, genetic data is concrete and not self-reported, so is thereby more consistent and reliable.</p>
<p>Alternatively, existing genetic research resources could be expanded in purpose. The UK Biobank database contains genetic, lifestyle and health data for half a million UK participants (albeit it could be stronger with <a href="https://doi.org/10.1038/s41586-024-07721-5">more family-based participants</a>). Estonia, Iceland and the Scandinavian countries have large genetic databases. These could provide experimental participants together with polygenic scores. Experimental data then forms part of the growing data resource. Commercial providers such as 23andMe with large customer databases could even seek alternative revenue sources by facilitating the provision of participants for experimental studies.</p>
<p>Critical to the value of any future data sets is representative population sampling. However, polygenic scores are typically developed from homogeneous population groups, most commonly with European ancestry. As a result, polygenic scores cannot simply be plugged into analyses of diverse groups. Genetic data itself doesn’t solve the WEIRD problem if research panels remain WEIRD. These are the important but ultimately tractable issues we should be grappling with.</p>
<hr>
<p>My PhD was on the link between human evolution and economic growth. When I presented a draft of my PhD research proposal, the first comment I received was that I should refuse any grants from men with funny little moustaches and straight-arm salutes. Although that commenter came around, this initial reaction is a typical response to a discussion of genetics in social science.</p>
<p>That ‘fear’ of the implications of genetics, however, is not the only obstacle to its use. Genetic data is simply not available for many studies. Its absence makes it easy to ignore. If the authors don’t mention genetics in their analysis, few peer reviewers will criticise them for overlooking an obvious confound. They can’t ask for further analysis when the data is not there.</p>
<p>The result is that, despite examples of the type I have discussed above, the potential for genetics to inform our thinking on important policy questions is untapped. I trawled policy-focused papers on social mobility in Australia, including <a href="https://www.google.com/search?q=https://s3.documentcloud.org/documents/23571805/intergenerational-income-mobility-in-australia.pdf">Treasury policy briefs</a>, reports by the <a href="https://www.google.com/search?q=https://www.aihw.gov.au/getmedia/37c2c8b7-328c-41e1-bace-87ed7a551777/australias-welfare-chapter-2-summary-18sept2019.pdf.aspx%23:~:text%3DThere%2520is%2520clear%2520evidence%2520that,most%2520notably%2520the%2520United%2520States%2529.">Australian Institute for Health and Welfare</a> and <a href="https://www.pc.gov.au/media-speeches/speeches/inequality-government-role">speeches by the head</a> of the Australian Government’s major economic think tank. Genetics does not get a mention. It is possible to dig up the <a href="https://insidestory.org.au/the-remarkable-persistence-of-power-and-privilege/">occasional left-of-centre politician</a> who realises genetics can affect social mobility (albeit that politician was an economics professor who published on social mobility). However, even they <a href="https://ministers.treasury.gov.au/ministers/andrew-leigh-2022/articles/opinion-piece-no-childs-future-should-be-pre-determined-their">are silent on genetics</a> when they move into political mode.</p>
<p>That said, it is hard for policymakers to engage with genetic questions when the research comes from longitudinal datasets without genetic data. Insightful papers examining the genetics of social mobility or other economic questions are rare. When faced with a particular policy question, it is unlikely that genetic analysis is available, especially one that matches relevant populations or precise policy measures.</p>
<p>That is why enriching our economic datasets is so important. A robust genetic data foundation is crucial for advancing our understanding of policy questions such as social mobility, inequality, skill development, and the diversity of our responses to government interventions. By enabling more studies that incorporate genetics, we can expose the limitations of research that ignores this critical factor. Only then can we answer vital questions and develop more effective policies.</p>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp("https:\/\/www\.jasoncollins\.blog");
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
            // target, if specified
            link.setAttribute("target", "_blank");
            if (link.getAttribute("rel") === null) {
              link.setAttribute("rel", "noopener");
            }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
<input type="hidden" id="giscus-base-theme" value="light">
<input type="hidden" id="giscus-alt-theme" value="light">
<script>
function loadGiscusWhenReady() {
  // Function to get the theme based on body class
  const getTheme = () => {
    const baseTheme = document.getElementById('giscus-base-theme').value;
    const altTheme = document.getElementById('giscus-alt-theme').value;
    return document.body.classList.contains('quarto-dark') ? altTheme : baseTheme;
  };
  // Create the Giscus script and add it to the desired location
  const loadGiscus = () => {
    const script = document.createElement("script");
    script.src = "https://giscus.app/client.js";
    script.async = true;
    script.dataset.repo = "jasonacollins/jasoncollinsblog";
    script.dataset.repoId = "R_kgDOIbjYgQ";
    script.dataset.category = "Announcements";
    script.dataset.categoryId = "DIC_kwDOIbjYgc4CSiuB";
    script.dataset.mapping = "pathname";
    script.dataset.reactionsEnabled = "0";
    script.dataset.emitMetadata = "0";
    script.dataset.inputPosition = "bottom";
    script.dataset.theme = getTheme();
    script.dataset.lang = "en";
    script.crossOrigin = "anonymous";
    // Append the script to the desired div instead of at the end of the body
    document.getElementById("quarto-content").appendChild(script);
  };
  // MutationObserver to detect when the 'quarto-light' or 'quarto-dark' class is added to the body
  const observer = new MutationObserver((mutations) => {
    for (const mutation of mutations) {
      if (mutation.type === "attributes" && mutation.attributeName === "class") {
        if (document.body.classList.contains('quarto-light') || document.body.classList.contains('quarto-dark')) {
          loadGiscus();
          observer.disconnect(); // Stop observing once Giscus is loaded
          break;
        }
      }
    }
  });
  // Start observing the body for class attribute changes
  observer.observe(document.body, {
    attributes: true,
    attributeFilter: ["class"],
  });
}
loadGiscusWhenReady();
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
    <a class="nav-link" href="https://creativecommons.org/licenses/by/4.0/">
<p>Copyright: CC-BY</p>
</a>
  </li>  
</ul>
    </div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




</body></html>